src/bundle/Controller/ContentViewController.php

```diff
@@ -72,8 +74,12 @@ public function detailsTabAction(ContentView $view)
         return $view;
     }
 
-    public function versionsTabAction(ContentView $view, VersionFilter $versionFilter)
-    {
+    public function versionsTabAction(
```

Comment:
I think it would be a good idea but this PR is already quite big and is holding up some of the other PR's. We would also need to refactor some of it because it uses this `$this->getTranslations` and `$this->loadUser`. Can it be done afterwards in a separate PR?